robot_calibration_3d/urdf/common_materials.xacro

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://wiki.ros.org/xacro">
+  <xacro:include filename="$(find fanuc_resources)/urdf/common_colours.xacro"/>
```

Comment:
Currently looking for the fanuc_resources package. Update to robot_calibration_3d